public/resources/scss/index.scss

```diff
@@ -3956,3 +3956,27 @@ body {
 .narrow-info-container.top-deaths {
     margin-bottom: 15px;
 }
+
+.piece.piece-rarity-upgrade-1 {
+    border: 0 dashed;
+    position: relative;
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 1rem;
```

Comment:
why is it sized in rems?

Reply:
I always use rems, it helps keeping a consistent styling between elements... here it doesn't really makes sense because everything else is in px but.... 🤷